install.sh

```diff
@@ -386,12 +390,21 @@ directories=(bin etc include lib sbin share opt var
              var/log var/homebrew var/homebrew/linked
              bin/brew)
 group_chmods=()
+
 for dir in "${directories[@]}"; do
   if exists_but_not_writable "${HOMEBREW_PREFIX}/${dir}"; then
     group_chmods+=("${HOMEBREW_PREFIX}/${dir}")
   fi
 done
 
+if [[ -n "${HOMEBREW_MULTI_USER-}" ]] && exists_but_not_group_writable "${HOMEBREW_PREFIX}/Homebrew"; then
+  group_chmods+=("${HOMEBREW_PREFIX}/Homebrew")
+fi
+
+if [[ -n "${HOMEBREW_MULTI_USER-}" ]]; then
+  directories+=(Homebrew)
```

Comment:
How should this be changed when `HOMEBREW_PREFIX = HOMEBREW_REPOSITORY`?

Reply:
Since `$HOMEBREW_REPOSITORY` is always under the `$HOMEBREW_PREFIX` hierarchy:
```
if [[ -n "${HOMEBREW_MULTI_USER-}" && "${HOMEBREW_PREFIX}" != ${HOMEBREW_REPOSITORY} ]]; then
  directories+=("${HOMEBREW_REPOSITORY#${HOMEBREW_PREFIX}/}")
fi
```